database/migrations/2015_03_23_002918_create_issues_table.php

```diff
@@ -14,7 +14,7 @@ public function up() {
 		Schema::create('issues', function (Blueprint $table) {
 			$table->increments('id');
 			$table->string('title');
-			$table->text('description');
+			$table->string('description')->nullable();
```

Comment:
Actually, now that I'm thinking about it, why not create a new migration file and alter the description column to have a default value instead? That way, existing installations can simply migrate instead of needing to migrate:refresh (which would remove existing data)